components/email/src/Email.svelte

```diff
@@ -684,31 +689,32 @@
 
   function handleDraftClick(event: MouseEvent, draft: Message) {
     event.stopImmediatePropagation();
-    dispatchDraftEvent(event, draft);
+    dispatchDraftClickEvent(event, draft);
   }
 
   function handleDraftKeypress(event: KeyboardEvent, draft: Message) {
     event.stopImmediatePropagation();
     if (event.code === "Enter") {
-      dispatchDraftEvent(event, draft);
+      dispatchDraftClickEvent(event, draft);
     }
   }
 
-  function dispatchDraftEvent(event: UIEvent, draft: Message) {
+  function dispatchDraftClickEvent(event: UIEvent, draft: Message) {
     if (draft) {
+      draft.draft_id = draft.id;
```

Comment:
did the API change? Why is there is a `draft_id` prop now?

Reply:
Hi @heathergh that's definitely something we need to include in our docs.
We need to use `draft_id` to tell `/send` api that we are sending an existing draft message.
So we don't need to delete draft message in frontend code as api will handle it.  [send.py](https://github.com/nylas/cloud-core/blob/9d8f891fa3dc26133ba0661a764a3e5b3f806778/nylas/services/api/blueprints/send.py#L91)